src/main/resources/application-deploy.yml

```
@@ -0,0 +1,13 @@
+  config:
+    activate:
+      on-profile: deploy
+
+  datasource:
+    url: ${SPRING_DATASOURCE_URL}
+    username: ${SPRING_DATASOURCE_USERNAME}
+    password: ${SPRING_DATASOURCE_PASSWORD}
+    driver-class-name: com.mysql.cj.jdbc.Driver
+  jpa:
+    hibernate:
+      ddl-auto: update
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)